README.rst

```diff
@@ -92,24 +89,38 @@ You can use the `django CMS installer <https://djangocms-installer.readthedocs.i
     (env) $ djangocms -f -p . my_demo
     (env) $ python manage.py
 
+****
+Demo
+****
+
+.. image:: https://raw.githubusercontent.com/divio/django-cms/develop/docs/images/try-with-divio.png
+   :target: http://try.django-cms.org/
+   :alt: Try demo with Divio Cloud
```

Comment:
@FinalAngel, is that the demo? At the first glance it appears to be the divio account creation?

Reply:
@victor-yunenko yes it is the demo, open it up in an incognito window :)

Reply:
I did, but after the signup it looks like I ended up on the empty projects page:

![image](https://user-images.githubusercontent.com/36416236/97416199-d349d580-190e-11eb-89af-72ceec421668.png)

Perhaps we should show there the one click demo creation page? Eg https://control.divio.com/demo/get-new/

Reply:
Well the redirect should probably be changed to https://control.divio.com/demo/get-new/ then. @victor-yunenko at best check that with @macolo maybe on the technical meeting?